index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="styles.css">
+    </head>
```

Comment:
Great job for using the <head> semantic tag, but you forget to include some important tag which is <meta>. using meta tags in the head section of an HTML file can provide important information about the page's content, help with search engine optimization, and ensure that the page is displayed correctly in different browsers and on different devices.